Admin/RuntimeListResponse.cs

```diff
@@ -67,5 +67,14 @@ public string Uri
             get;
             set;
         }
+        
+        /// <summary>
+        /// The version of the runtime the node is currently running.
+        /// </summary>
+        public string Version
```

Comment:
See comment from the flow PR about a more descriptive name

Reply:
Changed to LatestVersion and updated the description a bit to match.